src/WebJobs.Script/Workers/WorkerConcurrencyManager.cs

```diff
@@ -101,6 +128,52 @@ internal async void OnTimer(object sender, System.Timers.ElapsedEventArgs e)
             _timer.Start();
         }
 
+        private void OnActivationTimer(object sender, System.Timers.ElapsedEventArgs e)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            try
+            {
+                if (_timer == null && _scmHostingConfigurations.FunctionsWorkerDynamicConcurrencyEnabled)
```

Comment:
as mentioned above, if this can happen within ColdStartDelayMS then we might need to delay things a bit here to avoid coldstart impact. looks for ExecuteAfterColdStartDelay

Reply:
To activate worker concurrency first attempt to call GetValue will be in 5 minutes after function host start:
https://github.com/Azure/azure-functions-host/pull/8125/files#diff-e7382bebc5d5b35696a4fe7db2abd00ab8fb626b290713ee625de9f4fe0176a3R35